README.md

```diff
@@ -58,15 +58,15 @@ See the [contribution guidelines](https://github.com/nlohmann/json/blob/master/.
 The single required source, file `json.hpp` is in the `single_include/nlohmann` directory or [released here](https://github.com/nlohmann/json/releases). All you need to do is add
 
 ```cpp
-#include "json.hpp"
+#include "nlohmann/json.hpp"
```

Comment:
I am not sure about this. The paragraph before references the specific file, and the paragraph after that no additional headers are required.

Any opinions on this?

Reply:
I see your point and kind of agree with it. But I think it would be good to promote consistency and including the file as `nlohmann/json.hpp`.
Otherwise, for example for a beginner, it would most likely be quite a bit confusing.
I would maybe argue that people who download just the header know what they're doing and "are on their own" with regards to include directory and making sure that the file `json.hpp` is unique etc. And for everybody else, the readme should show the prefered, "namespaced" way to include the library, via `nlohmann/json.hpp`.

But I think you can probably make the argument both ways ;-)

Reply:
In fact, all examples in `doc/examples` use `#include <nlohmann/json.hpp>` and accompanied with a note to compile with `-Isingle_include`. So the README should be consistent to this, too.

Reply:
Could we also change the include style? By using angle brackets instead of double quotes?

This is a minor thing, but it could avoid some confusion about the location of the file

Reply:
@theodelrieu I'm personally using angle-brackets for system includes and double quotes for "local"/third-party includes, but I'm happy to stick to the conventions of this repo ;-)